lxd/daemon.go

```diff
@@ -1823,14 +1823,8 @@ func (d *Daemon) setupRBACServer(rbacURL string, rbacKey string, rbacExpiry int6
 		return nil
 	}
 
-	// Get a new server struct
-	server, err := rbac.NewServer(rbacURL, rbacKey, rbacAgentURL, rbacAgentUsername, rbacAgentPrivateKey, rbacAgentPublicKey)
-	if err != nil {
-		return err
-	}
-
 	// Set projects helper
-	server.ProjectsFunc = func() (map[int64]string, error) {
+	getProjects := func() (map[int64]string, error) {
```

Comment:
What was the reason for the intermediate variable here?

Reply:
I've changed the code in a way that this function is created inside the rbac server rather than defining it outside of the package.